PixelHunter1995/Content/SmurphFiles/dialog_intro.smph

```diff
@@ -0,0 +1,95 @@
+#IntroInit
+Nick: OK, so I've started the boiler and poured up the cooking oil.
+      What's the next ingredient?
+Ursula: Let's see here...
+        Next, the recipe tells you to add charcoal, iron and salt.
+Nick: Charcoal, iron and salt, check! Felixia, could you fetch those for me?
+Felixia: Um, OK...
+<goto IntroNickOptions>
+
+#IntroNick
+Ursula: Hey Nick, how's it bubbling?
+Nick: Could you get the remaining ingredients, please?
+<goto IntroNickOptions>
+
+#IntroNickOptions
+- [!found_salt] Where can I find salt?
+  Felixia: Where can I find salt?
+  Nick: There should probably be something around here containing salt.
+- [!found_charcoal] Where can I find charcoal?
+  Felixia: Where can I find charcoal?
+  Nick: No idea.
+- [!found_iron] Where can I find iron?
```

Comment:
We also need a way to distinguish what happens here. It's not clear that all of the options belong to the same node.

Reply:
With gotos now, all options in a list will belong to the same node.